Composer/packages/client/src/components/GetStarted/GetStarted.tsx

```diff
@@ -4,148 +4,75 @@
 /** @jsx jsx */
 import { jsx } from '@emotion/core';
 import React from 'react';
-import { useRecoilValue } from 'recoil';
 import formatMessage from 'format-message';
-import { Stack } from 'office-ui-fabric-react/lib/Stack';
-import { Link } from 'office-ui-fabric-react/lib/Link';
-import { Panel } from 'office-ui-fabric-react/lib/Panel';
+import { Panel, IPanelStyles } from 'office-ui-fabric-react/lib/Panel';
+import { Pivot, PivotItem } from 'office-ui-fabric-react/lib/Pivot';
 
-import { localBotsDataSelector } from '../../recoilModel/selectors/project';
-import { currentProjectIdState, locationState } from '../../recoilModel';
-import TelemetryClient from '../../telemetry/TelemetryClient';
-
-import { h3Style, ulStyle, liStyle } from './styles';
+import { GetStartedNextSteps } from './GetStartedNextSteps';
+import { GetStartedLearn } from './GetStartedLearn';
 
 type GetStartedProps = {
   isOpen: boolean;
+  showTeachingBubble: boolean;
+  requiresLUIS: boolean;
```

Comment:
Nit: I see a lot of specific recognizer flags.  We should consider having things like requiredRecognizers: string[], but it would have to be across the whole codebase.

Reply:
This is about requiring the Azure resource _related_ to the recognizer, which right now is a pretty fixed list. HOWEVER, I agree, it would be nice for recognizers to automatically manifest the resources they require and populate this and other places.

one day!